### PR TITLE
updating versions

### PR DESCRIPTION
**Pull request by jgermade**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ng-lodash",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (nwhite89) [inline bower.json]: Can we update this to 0.2.0 please it has a potential of breaking changes and I'd like to not enforce this upon those that don't know about it thanks.

File: package.json [hunk 2]
```diff
@@ -27,17 +27,17 @@
     "grunt": "~0.4.5",
     "grunt-bower-task": "~0.4.0",
     "grunt-cli": "~0.1.13",
-    "grunt-contrib-jshint": "~0.10.0",
-    "grunt-contrib-uglify": "~0.5.1",
-    "grunt-jscs": "~0.8.1",
-    "grunt-karma": "~0.9.0",
-    "grunt-lodash": "~0.3.0",
+    "grunt-contrib-jshint": "^0.10.0",
```
Comment (nwhite89) [inline package.json]: can you change these back to ~ the problem about using ^ it enforces larger changes than simple minor changes which may cause error's unknown to the version we believe to be running thanks. If you'd like to update them to the latest version numbers that is not a problem. Thanks

**Comment by jgermade** [comment]
Hi @nwhite89, what about now?

**Comment by nwhite89** [comment]
great thanks for your work @jgermade

**Comment by jgermade** [comment]
is a pleasure can contribute

**Pull request merged**